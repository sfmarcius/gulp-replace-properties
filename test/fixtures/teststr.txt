{greet} javascript!
{greet} java!
{greet} node!
