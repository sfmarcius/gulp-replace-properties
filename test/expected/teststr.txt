Hello javascript!
Hello java!
Hello node!
